test/e2e/v1beta1/placement_test.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package v1beta1
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	placementv1beta1 "go.goms.io/fleet/apis/placement/v1beta1"
+)
+
+var _ = Describe("placing resources using a CRP with no placement policy specified", Serial, Ordered, func() {
```

Comment:
is there any reason that those tests are serial and orderd?

Reply:
Will refactor the tests + enable parallelization. The original goal was to re-use some resources, plus the fact that we actually have never enabled parallelization in our E2E runs.